src/validators/__tests__/helpers.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { validatePassword } from '../helpers'
+
+describe('test validatePassword function', () => {
+  it('should return TRUE for validatePassword("Password123")', function () {
```

Comment:
```suggestion
  it('should return TRUE if password contains latin characters and numbers', function () {
```

Reply:
Thanks! I just edit code according to this comment in next [PR](https://github.com/what-the-hub/vue-game/pull/43/commits/81c59eeda875f83180b46d58801bd36fc53c68c9)